test/parse.js

```diff
@@ -404,3 +404,96 @@ test('query strings having (:list) colon-list-separator arrays', t => {
 test('query strings having (:list) colon-list-separator arrays including null values', t => {
 	t.deepEqual(queryString.parse('bar:list=one&bar:list=two&foo', {arrayFormat: 'colon-list-separator'}), {bar: ['one', 'two'], foo: null});
 });
+
+test('pre-defined types parsing: can override a parsed number to be a string ', t => {
```

Comment:
```suggestion
test('types option: can override a parsed number to be a string ', t => {
```

same with the others